src/functionModules/components/FileUploaderModule.ts

```diff
@@ -58,6 +56,11 @@ export default class FileUploaderModule {
   }
 
   public static ethereumDisabled(): boolean {
-    return typeof ethereum === 'undefined' || typeof ethereum.selectedAddress === 'undefined'
+    return typeof ethereum === 'undefined' ||
+      ethereum === null ||
+      typeof ethereum.selectedAddress === 'undefined' ||
+      ethereum.selectedAddress === null ||
+      typeof ethereum.selectedAddress !== 'string' ||
+      ethereum.selectedAddress.length <= 0
```

Comment:
can be `return !!!ethereum || !!!ethereum.selectedAddress || typeof ethereum.selectedAddress !== 'string'`